src/commands/ScopeCommand.java

```diff
@@ -26,27 +26,34 @@ public void execute(String[] input, SimulationManager simulationManager) {
         }
 
         if (input.length == 1) {
-            System.out.println();
             Map<String, List<String>> scopes = simulationManager.getScopes();
-            for (String label : scopes.keySet()) {
-                StringBuilder sb = new StringBuilder();
-                sb.append(String.format("%s: %s ", label, AlloyConstants.BLOCK_INITIALIZER));
-                sb.append(String.format("%s %s", String.join(", ", scopes.get(label)), AlloyConstants.BLOCK_TERMINATOR));
-                System.out.println(sb.toString());
-            }
-            System.out.println();
+            System.out.println(stringForScopeSet(scopes));
             return;
         }
         String sigName = input[1];
         List<String> scope = simulationManager.getScopeForSig(sigName);
-        if (scope == null) {
-            System.out.println(CommandConstants.SIG_NOT_FOUND);
-            return;
+        System.out.println(stringForSig(scope));
+    }
+
+    private String stringForScopeSet(Map<String, List<String>> scopes) {
```

Comment:
Nit: this function name is a bit of a misnomer since `scopes` isn't a set. I would call this something like `formattedScopes` for example.